src/mocks/handlers.ts

```diff
@@ -3,7 +3,16 @@ import API_ENDPOINTS from "src/config/api";
 import mockFileSubmissions from "src/mocks/data/fileStatus.json";
 
 export const handlers = [
-  http.get(API_ENDPOINTS.fileSubmissions, () => {
+  http.get(API_ENDPOINTS.fileSubmissions, ({ request }) => {
+    const url = new URL(request.url);
+    const dataStreamId = url.searchParams.get("data_stream_id");
+
+    // bad request wihtout data_stream_id
```

Comment:
Nit: I don't think this comment is necessary. Also, there's a typo in without.

Reply:
yea, these 2 comments could come out